### ISSUE TITLE
Epic: Send flu results to ReportStream

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4439]
ReportStream has asked us to begin sending them flu results (see https://skylight-hq.slack.com/archives/C01LTSNKEPP/p1664902630960239 for details).

This epic tracks the work required to send a separate feed of flu results to ReportStream. This will entail at minimum:
- Converting flu results to JSON (using TestEventExport or similar)
- Putting those JSONs on a queue somewhere, either the same Azure storage queue or a new one
- Sending the flu results in batch to ReportStream as a CSV, the same way we do for covid results. The flu results will need to be in a different CSV file though, and will likely be sent to a different endpoint 

Further research/decision-making is needed before we can take further action on this. Specifically, we need to determine:
- Will we use the same Azure function application for flu that we have for sending COVID-19 results?
- Will we use the same storage queue, or do we need to set up a new one?

**Comment by emmastephenson** [comment]
Superseded by #4738, Multiplex on FHIR.
